portfolio/src/main/java/com/google/sps/data/User.java

```diff
@@ -0,0 +1,14 @@
+package com.google.sps.data;
+
+/** Individual user data. */
+public final class User {
+  private final boolean isLoggedin;
+  private final String email;
+  private final String url;
```

Comment:
what is the url for? it's not immediately obvious why a User needs a url

Reply:
If the user is logged in, the servlet will create a logout URL; vice versa.

Reply:
can you add a comment explaining what the URL is?

Reply:
Done